### PR TITLE
Update RadioButton and Checkbox Components

### PR DESCRIPTION
**Pull request by bankai254**
- Add __defaultChecked__ prop for RadioButton 
- Add __displayStyle__ option for RadioButton and Checkbox to toggle between *inline* and *stacked* layouts
- Add Notes for RadioButton and Checkbox
- Add displayStyle spacing default values to cobalt and default themes

@ben-dalton @Dantiff @tdakhla @hamholla

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @tdakhla, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket](https://technekes.atlassian.net/browse/TPMTI-99)
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Checkbox/Checkbox.component.tsx [hunk 1]
```diff
@@ -53,11 +59,26 @@ export type Props = {
...
 
-const SDiv = styled.div`
-  display: inline-block;
-  + div[data-radiobutton] {
-    padding-left: 20px;
+const displayStyleRules: Function = (
+  displayStyle: 'inline' | 'stacked',
+  theme: any,
+) => {
+  console.log(theme);
```
Comment (tdakhla) [inline src/Checkbox/Checkbox.component.tsx]: console.log should be removed.

File: src/Checkbox/Checkbox.component.tsx [hunk 2]
```diff
@@ -53,11 +59,26 @@ export type Props = {
...
+  if (displayStyle === 'inline') {
+    return {
+      display: 'inline-block',
+      'padding-right': theme.radioButton.inlineSpacing,
```
Comment (tdakhla) [inline src/Checkbox/Checkbox.component.tsx]: This should be `theme.checkbox.inlineSpacing`.

File: src/RadioButton/RadioButton.component.tsx [hunk 3]
```diff
@@ -56,10 +67,26 @@ export type Props = {
...
+  console.log(theme);
+  if (displayStyle === 'inline') {
+    return {
+      display: 'inline-block',
+      'padding-right': theme.radioButton.inlineSpacing,
+    };
+  }
+  return {
+    display: 'block',
+    'padding-bottom': theme.checkbox.stackedSpacing,
```
Comment (tdakhla) [inline src/RadioButton/RadioButton.component.tsx]: This should be `theme.radioButton.stackedSpacing`.

**Comment by tdakhla** [comment]
Just a few minor things, but other than that it looks good 👍

File: src/RadioButton/RadioButton.component.tsx (hunk 3 above)
Comment (bankai254) [reply src/RadioButton/RadioButton.component.tsx]: Fixed

File: src/Checkbox/Checkbox.component.tsx (hunk 1 above)
Comment (bankai254) [reply src/Checkbox/Checkbox.component.tsx]: Fixed

File: src/Checkbox/Checkbox.component.tsx (hunk 2 above)
Comment (bankai254) [reply src/Checkbox/Checkbox.component.tsx]: Fixed

**Comment by bankai254** [comment]
@tdakhla All items have been fixed

**Pull request merged**